Assets/MixedRealityToolkit.Tests/PlayModeTests/NearInteractionTouchableTests.cs

```diff
@@ -421,6 +421,150 @@ public IEnumerator NearInteractionTouchableUnityUiToggle()
             UnityEngine.Object.Destroy(canvas.gameObject);
         }
 
+        /// <summary>
+        /// Test minimum touch distance.
+        /// </summary>
+        /// <remarks>
+        /// Near interaction uses two separate distance calculations:
+        /// 1. The PokePointer uses the DistanceToTouchable to determine the closest touchable. This distance value is also used compared to PokePointer.TouchableDistance to determine the set of eligible touchables in the first place.
+        ///     There is no explicit spec which space this distance is computed in, although it should be in world space. Due to the way touchables convert pointer position into local space first and then do a scale-only inverse transform there may be errors.
+        /// 1. After the PokePointer has selected the closest touchable, it then performs a raycast against either the collider or the UnityUI canvas, depending on type. The (world space) length of that ray is then used as the actual distance for triggering touch events.
+        ///     When the ray length is _less_ than the PokeThreshold the TouchDown event is raised and the touchable is "down".
+        ///     When the touchable is "down" and the ray length is _greater_ than PokeThreshold + DebounceThreshold (!) the TouchUp event is raised and the touchable is released.
+        /// 
+        /// The normal vector for performing the ray cast, however, is still the one returned by the first distance calculation.
+        /// 
+        /// NearInteractionTouchable also calculates distance from the object center, but the raycast ignores this and uses distance from the collider.
+        /// UnityUI OTOH does a raycast against a flat canvas, so the two distance values _should_ match (but it's not guaranteed).
+        /// 
+        /// Default PokeThreshold is larger than the TouchableDistance, so we have to reduce it in the tests below to avoid dropping the active touchable before exceeding the DebounceThreshold.
+        /// 
+        /// The ray in the second step starts at Position - PokeThreshold, but this is not taken into account when raising events. So effectively one has to move the pointer right up to the collider in order to trigger a touch, regardless of the PokeThreshold. This may be intentional or not, it's not documented at all.
```

Comment:
The expected behavior is to not need to move right up to the collider unless PokeThreshold is zero. If PokeThreshold was 0.1, then the touch should start at distance 0.1 away

Reply:
Shouldn't be hard to fix, just have to add the right offset in the distance calculation in PokePointer. There just wasn't any comment in the code that indicated the intended behavior.

Reply:
I have fixed the poke pointer distance calculation now and updated the tests. Seems to work fine in the test scene too.